src/utests/utf_baselib_jni/UtfBaselibJniMain.cpp

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
The JNI code and respectively tests should be enabled only if JDK / JRE is installed / enabled in the build environment... in general should be optional dependency (opt-in)... and only there if used